Automattic/pocket-casts-ios
# Revert: hidden chapters

With the changes made in #553 we had a regression in which podcasts that used to show chapters didn't display them anymore (eg.: Tiesto - Clublife). That includes showing the "Chapters" tab and all related UI stuff (progress of chapter, arrows, etc).

This PR reverts this change to restore the previous feature while we fix it.

## To test

1. Run the app
2. Search for Clublife
3. Play any episode
4. ✅ You should see the "Chapters" tab
5. Tap on it
6. ✅ You should see the current chapter progress

## Checklist
